### ISSUE TITLE
Dropout is too low for the TED dataset

### ISSUE THREAD

**Issue by kdavis-mozilla** [mozilla/DeepSpeech#336]
With dropout 10% at 20 epochs we have a training WER of 15.9%
```
2017-01-06 17:18:56 STARTING Epoch 0019
2017-01-06 17:18:56 Training model...
2017-01-06 20:37:09 Training loss=12.977073401 avg_cer=0.039520074 WER: 0.159389
2017-01-06 20:37:09 --------------------------------------------------------------------------------
2017-01-06 20:37:09     WER:    0.076923
2017-01-06 20:37:09     loss:   0.843824
2017-01-06 20:37:09     source: "and this is what we mean by multiple and interacting causes of death"
2017-01-06 20:37:09     result: "and this is what we mean by multiple and interacting cases of death"
2017-01-06 20:37:09 --------------------------------------------------------------------------------
2017-01-06 20:37:09     WER:    0.142857
2017-01-06 20:37:09     loss:   0.800132
2017-01-06 20:37:09     source: "i discovered something about love and design"
2017-01-06 20:37:09     result: "i discoverd something about love and design"
2017-01-06 20:37:09 --------------------------------------------------------------------------------
...
```
while at 20 epochs our validation WER is 51.2%
```
2017-01-06 20:37:16 Validating model...
2017-01-06 20:37:52 Validation loss=178.301272392 avg_cer=0.212876620 WER: 0.512488
2017-01-06 20:37:52 --------------------------------------------------------------------------------
2017-01-06 20:37:52     WER:    0.166667
2017-01-06 20:37:52     loss:   2.680958
2017-01-06 20:37:52     source: "is this good news or bad news and the answer is yes"
2017-01-06 20:37:52     result: "is this good news or badnows and the answer is yes"
2017-01-06 20:37:52 --------------------------------------------------------------------------------
2017-01-06 20:37:52     WER:    0.352941
2017-01-06 20:37:52     loss:   9.723542
2017-01-06 20:37:52     source: "in twenty five years its gone from this to this this is a problem because the warming"
2017-01-06 20:37:52     result: "into wenty five years its gone from this tothis this is a problem because thewarming"
2017-01-06 20:37:52 --------------------------------------------------------------------------------
...
```
To narrow the training vs validation WER difference the dropout rate should be increased.

Here we increase the dropout rate from 10% to 20%

**Comment by andreasgal** [comment]
Any idea why the results of the paper don't reproduce here?

**Comment by kdavis-mozilla** [comment]
@andreasgal My guess is that's it's a result of us using LSTM's for the recurrent layer and Baidu used RNN's.

The LSTM's are better at fitting time series than RNN's. Thus the LSTM's overfit more effectively than the RNN's. Thus the LSTM'S require a higher dropout.
